Source/ACE.Server/Managers/LandblockManager.cs

```diff
@@ -212,5 +220,28 @@ public static void UnloadLandblocks()
                 }
             }
         }
+
+        public static void Initialize()
+        {
+            var landBlockIdList = new List<LandblockId>();
+
+            for (uint x = 0; x < RawLandblockId.Length; x++)
+            {
+                var landBlockId = new LandblockId(RawLandblockId[x]);
+                landBlockIdList.Add(landBlockId);
+            }
+
+            foreach (var landBlockId in landBlockIdList)
+            {
+                Stopwatch sw = Stopwatch.StartNew();
+                ForceLoadLandBlock(landBlockId, true);
+                sw.Stop();
+                log.DebugFormat("Landblock {0:X4} preloaded in {1} milliseconds", landBlockId.Landblock, sw.ElapsedMilliseconds);
+            }
+        }
+
+        // TODO: Change the RawLandblockId list used for preloading defined landblocks to some other, more easily-modified format, rather than a compiled uint array
+        private static readonly uint[] RawLandblockId = { 0xa9b4ffff, 0xabb2ffff, 0xaab3ffff, 0x7d64ffff, 0x7e64ffff, 0xe64effff, 0xe74effff, 0xda55ffff, 0xdb54ffff,
```

Comment:
Can this be changed so that each id is on it's own line followed by a comment describing the landblock by name?